src/app/shell/info/support/support.component.html

```diff
@@ -1,29 +1,31 @@
-<div *ngIf="(platformSupport$ | async)?.companyInformationItems.length; else noInfo">
-  <div class="support-content">
-    <div>
-      <h2 class="support-title">{{ (platformSupport$ | async).title }}</h2>
-    </div>
-    <div class="support-text">
-      <div *ngFor="let item of (platformSupport$ | async)?.companyInformationItems">
-        <p><strong>{{ item.sectionName }}</strong></p>
-        <p>{{ item.description }}</p>
+<ng-container *ngIf="!(isLoading$ | async)">
+  <div *ngIf="(platformSupport$ | async)?.companyInformationItems.length; else noInfo">
+    <div class="support-content">
+      <div>
+        <h2 class="support-title">{{ (platformSupport$ | async).title }}</h2>
+      </div>
+      <div class="support-text">
+        <div *ngFor="let item of (platformSupport$ | async)?.companyInformationItems">
+          <p><strong>{{ item.sectionName }}</strong></p>
+          <p>{{ item.description }}</p>
+        </div>
+        <p class="support-contact"> У разі технічних питань писати на:
+          <a class="support-link" href="mailto:pozashkillia@gmail.com" target="_blank">
+            pozashkillia@gmail.com
+          </a>
+        </p>
+        <p class="support-contact"> У разі будь яких інших питань звертатися на:
+          <a class="support-link" href="mailto:pozashkillyainfo@gmail.com" target="_blank">
+            pozashkillyainfo@gmail.com
+          </a>
+        </p>
+        <p class="support-contact"> Або за телефоном:
+          <a class="support-link" href="tel:+380977777777" alt="Телефон +380977777777">+38(097) 777 7 777</a>
+        </p>
```

Comment:
we agreed to remove this static sections

Reply:
removed